### ISSUE TITLE
Production server can't find templates

### ISSUE THREAD

**Issue by vmihailenco** [mintchaos/django_compressor#45]
I have issue that exactly match to this one:
http://github.com/dziegler/django-css/issues/closed#issue/10

On dev server (windows) all work well, but on production I get an exception:

```
Exception: TemplateDoesNotExist: compressor/css.html
```

With COMPRESS = False I don't get this exception.

easy_install output:

```
Searching for django-compressor
Reading http://pypi.python.org/simple/django_compressor/
Reading http://github.com/mintchaos/django_compressor
Best match: django-compressor 0.5.3
Downloading http://pypi.python.org/packages/source/d/django_compressor/django_compressor-0.5.3.tar.gz#md5=c211a9d5876b587777c8443846265770
Processing django_compressor-0.5.3.tar.gz
Running django_compressor-0.5.3/setup.py -q bdist_egg --dist-dir /tmp/easy_install-42zErv/django_compressor-0.5.3/egg-dist-tmp-gC3OmF
zip_safe flag not set; analyzing archive contents...
Adding django-compressor 0.5.3 to easy-install.pth file

Installed /nfs/http4/lout/modules/django_compressor-0.5.3-py2.6.egg
Processing dependencies for django-compressor
Finished processing dependencies for django-compressor
```

**Comment by diox** [comment]
vladimir-webdev: what do your TEMPLATE_LOADERS look like ?

**Comment by vmihailenco** [comment]
I have default template loaders for 1.2 version.

This problem happens because this app is not zip_safe, so it can not be installed as zipped egg.

**Comment by statico** [comment]
I'm confused. Is this an issue because django-compressor was installed as a zip archive but the environment doesn't allow it? (i.e., `django.template.loaders.eggs.load_template_source` can't be used)

**Comment by vmihailenco** [comment]
Yes, you are right.

**Comment by EmilStenstrom** [comment]
This could easily be worked around by copying the template file to your own project directory. compressor will detect your own template file that and use that instead.

**Comment by jezdez** [comment]
Prevent accidentally installing the app as an egg file and automatically install BeautifulSoup. Closed by b2e7725f2cb57d167b02f9a754f9604e5539ec81 and GH-58.
